openfisca_aotearoa/tests/demographics/relationships.yaml

```diff
@@ -1,23 +1,23 @@
 - name: Relationship status
-  period: "2018-10"
+  period: "2018-W40"
   absolute_error_margin: 0
   input:
     persons:
       Mama:
         age:
-          "2017-10-01": 30
+          "day:2017-10-01:7": 30
```

Comment:
Do we need these age variable periods to match the test period `period: "2018-W40"`?
eg.
```
  age:
    "day:2018-10-15:7": 30
```

Reply:
In fact, you can give a week, and it will pan over the 7 days.